editor/code_editor.cpp

```diff
@@ -650,12 +650,12 @@ void CodeTextEditor::_text_editor_gui_input(const Ref<InputEvent> &p_event) {
 }
 
 void CodeTextEditor::_zoom_in() {
-	font_resize_val += EDSCALE;
+	font_resize_val += EDSCALE < 1.0f ? 1 : EDSCALE;
```

Comment:
You change that by `MAX(EDSCALE, 1.0f)` (and also down there) ? 
That would make the code easier to read.